src/utils/format-date.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Display date
+ * Override browser timezone to always display same date as in data
+ */
+export type DateFormat =
+  | 'short' // 12/20
+  | 'long'; // Dec 20, 2020
+
+export function formatDate(
+  date: Date | undefined,
+  format: DateFormat = 'short',
+  locale: string = 'en-US'
+): string {
+  // Return blank if undefined
+  if (!date) return '';
+
+  const options: any = {
```

Comment:
You can use `Intl.DateTimeFormatOptions` instead of `any` so you get type safety.

Reply:
Done